app/src/androidTest/java/com/droidhats/campuscompass/views/MapFragmentTest.kt

```diff
@@ -1,5 +1,6 @@
 package com.droidhats.campuscompass.views
 
+import androidx.drawerlayout.widget.DrawerLayout
```

Comment:
On my end, this shows as an unnecessary import. I tested the code without this line and it passes, you can remove this

Reply:
good catch, since I ended up using DrawerActions, no need for DrawerLayout